Slambook/index.js

```diff
@@ -67,9 +67,32 @@ form.addEventListener('submit',function(e){
     allSlamArray.push(formValue)
     console.log(formValue)
 console.log(allSlamArray)
```

Comment:
Consider adding a comment or description above the function definition to explain its purpose or functionality.